{{cookiecutter.project_slug}}/HOWTO.md

```diff
@@ -5,25 +5,43 @@
 You may open your favorite IDE and start working with your project, please note that this project runs using docker.
 You may modify at any time the credentials used to authenticate to connect modifying the file:
 
-*{{ cookiecutter.project_slug }}/.{{ cookiecutter.project_slug }}_dev.env*
-
+    *{{ cookiecutter.project_slug }}/.{{ cookiecutter.project_slug }}_dev.env*
+
+You may add environment variables used by the extension on the descriptor file `extension.json`:
+
+    "variables":[
+        {
+            "name": "SECURE_VAR",
+            "secure": true
+        },
+        {
+            "name": "NOT_SECURE_VAR",
+            "value": "https://example.com",
```

Comment:
i think shall not be value, but rather default_value.

this is only used to create the var on first time and even populate if default is suggested, but not to cover the life cycle